CMakeLists.txt

```diff
@@ -11,14 +11,15 @@ endif()
 project("openscap")
 set(OPENSCAP_VERSION_MAJOR "1")
 set(OPENSCAP_VERSION_MINOR "3")
-#TODO: Remove this before final 1.3.0
-#FIXME: Windows Installer allows only integers in version name.
-if(WIN32)
-	set(OPENSCAP_VERSION_PATCH "0")
+set(OPENSCAP_VERSION_PATCH "0")
+set(OPENSCAP_VERSION_SUFFIX "alpha2")
+
+if(OPENSCAP_VERSION_SUFFIX AND NOT WIN32)
+	set(OPENSCAP_VERSION "${OPENSCAP_VERSION_MAJOR}.${OPENSCAP_VERSION_MINOR}.${OPENSCAP_VERSION_PATCH}.${OPENSCAP_VERSION_SUFFIX}")
 else()
-	set(OPENSCAP_VERSION_PATCH "0_alpha2")
+	set(OPENSCAP_VERSION "${OPENSCAP_VERSION_MAJOR}.${OPENSCAP_VERSION_MINOR}.${OPENSCAP_VERSION_PATCH}")
```

Comment:
@redhatrises With introduction of suffix, can't the versions be the same for win and linux? 
As in the CMakeLists it only fetches up to the [third token](https://github.com/OpenSCAP/scap-workbench/blob/v1-2/CMakeLists.txt#L39), I believe the suffix should be ignored by scap-workbench.

Reply:
@yuumasato not for Windows. It requires integers only for versioning.